pkgs/development/python-modules/django-cms/default.nix

```
@@ -0,0 +1,102 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  django,
+  pythonOlder,
+  pytestCheckHook,
+  setuptools,
+  django-classy-tags,
+  django-formtools,
+  django-treebeard,
+  django-sekizai,
+  djangocms-admin-style,
+  python,
+  dj-database-url,
+  djangocms-text-ckeditor,
+  fetchpatch,
+  django-cms,
+  gettext,
+  iptools,
+  ruff,
+}:
+
+buildPythonPackage rec {
+  pname = "django-cms";
+  version = "4.1.3";
+  pyproject = true;
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchFromGitHub {
+    owner = "django-cms";
+    repo = "django-cms";
+    rev = "refs/tags/${version}";
+    hash = "sha256-ucGxc6f2UYdmSLuyxVV2nIcl35qoeQMlk1paN6XeBFY=";
+  };
+
+  patches = [
+    # Removed django-app-manage dependency by updating ./manage.py
+    # https://github.com/django-cms/django-cms/pull/8061
+    (fetchpatch {
+      url = "https://github.com/django-cms/django-cms/commit/3270edb72f6a736b5cb448864ce2eaf68f061740.patch";
+      hash = "sha256-DkgAfE/QGAXwKMNvgcYxtO0yAc7oAaAAui2My8ml1Vk=";
+      name = "remove_django_app_manage_dependency.patch";
+    })
+    (fetchpatch {
+      url = "https://github.com/django-cms/django-cms/pull/8061/commits/04005ff693e775db645c62fefbb62367822e66f9.patch";
+      hash = "sha256-4M/VKEv7pnqCk6fDyA6FurSCCu/k9tNnz16wT4Tr0Rw=";
+      name = "manage_py_update_dj_database_url.patch";
+    })
+  ];
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    django
+    django-classy-tags
+    django-formtools
+    django-treebeard
+    django-sekizai
+    djangocms-admin-style
+  ];
+
+  nativeCheckInputs = [
+    gettext
+    pytestCheckHook
+  ];
+
+  checkInputs = [
+    dj-database-url
+    djangocms-text-ckeditor
+    iptools
+    ruff
```

```suggestion
    dj-database-url
    djangocms-text-ckeditor
    iptools
```
We don't need `pytestCheckHook` since we use custom checkPhase.
`ruff` is a formatter, so we don't want to run it in tests.